modules/jabba/1.0/src/R/make_germline_filter.R

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env Rscript
+
+script_path <- normalizePath(dirname(unlist(strsplit(commandArgs()[grepl('--file', commandArgs())], split = '='))[2]))
+
+args <- commandArgs(trailingOnly = TRUE)
+
+nrml.tbl.path <- args[1]
```

Comment:
Snakemake parses dictionary with all elements of the rule available directly. They can be accessed directly by name (`snakemake@input[["tbl"]]`) or by index (`snakemake@input[[2]]`). This might make scripts a bit easier to write. Just noting it here in case you are not aware of that feature.